docs/requirements.txt

```diff
@@ -1,6 +1,7 @@
 sphinx_rtd_theme
 sphinx-autobuild
 sphinx
+sphinxcontrib-autoyaml
```

Comment:
Please add this to the optional_requirements.json file as well under documentation

Reply:
do you mean requirements.py? I don't know about/see optional_requirements.json

Reply:
yeah...in 6 there was optional_requirements. but not any longer

Reply:
done